### DISCUSSION TITLE
Avalonia 11.0.7 Windows 7 64 is not supported？

### DISCUSSION THREAD

**Discussion by qfmee** [AvaloniaUI/Avalonia#14431] [Q&A]
Avalonia 11.0.7 Windows 7 64 is not supported！but versions before 11.0.6 support it！The program cannot be started. What remedial measures are there?

**Comment by maxkatz6** [comment]
Windows 7 isn't supported since January 14, 2020.

Still, Avalonia unofficially supported it until it was broken in 11.0.7.
It can be fixed relatively easily, if anybody wants to try: https://github.com/AvaloniaUI/Avalonia/issues/14309#issuecomment-1903866901
